components/eventing-controller/README.md

```diff
@@ -14,82 +14,80 @@ This component contains controllers for various CustomResourceDefinitions relate
-    | WEBHOOK_CLIENT_SECRET  | The Client Secret used by webhooks to acquire Access Tokens from Kyma.          | BEB     |
-    | WEBHOOK_TOKEN_ENDPOINT | The Kyma public endpoint to provide Access Tokens.                              | BEB     |
-    | DOMAIN                 | The Kyma cluster public domain.                                                 | BEB     |
-    | NATS_URL               | The URL for the NATS server.                                                    | NATS    |
-    | APP_LOG_FORMAT         | The format of the Application logs.                                             | Both    |
-    | APP_LOG_LEVEL          | The level of the Application logs.                                              | Both    |
+| Environment Variable          | Description                                                                     |
+| ----------------------------- | ------------------------------------------------------------------------------- |
+| **For both**                  |                                                                                 |
+| `APP_LOG_FORMAT`              | The format of the Application logs.                                             |
+| `APP_LOG_LEVEL`               | The level of the Application logs.                                              |
+| `BACKEND_CR_NAMESPACE`        | The namespace of the Backend Resource (CR).                                     |
+| `BACKEND_CR_NAME`             | The name of the Backend Resource (CR).                                          |
+| `PUBLISHER_IMAGE`             | The image of the Event Publisher Proxy.                                         |
+| `PUBLISHER_IMAGE_PULL_POLICY` | The pull-policy of the Event Publisher Proxy.                                   |
+| `PUBLISHER_PORT_NUM`          | The port number of the Event Publisher Proxy itself.                            |
+| `PUBLISHER_METRICS_PORT_NUM`  | The port number of the Event Publisher Proxy metrics.                           |
+| `PUBLISHER_SERVICE_ACCOUNT`   | The service account of the Event Publisher Proxy.                               |
+| `PUBLISHER_REPLICAS`          | The number of replicas of the Event Publisher Proxy.                            |
+| `PUBLISHER_REQUESTS_CPU`      | The CPU requests of the Event Publisher Proxy.                                  |
+| `PUBLISHER_REQUESTS_MEMORY`   | The memory requests of the Event Publish Proxy.                                 |
+| `PUBLISHER_LIMITS_CPU`        | The CPU limits of the Event Publisher Proxy.                                    |
+| `PUBLISHER_LIMITS_MEMORY`     | The memory limits of the Event Publisher Proxy.                                 |
+| **For NATS**                  |                                                                                 |
+| `NATS_URL`                    | The URL for the NATS server.                                                    |
```

Comment:
`EVENT_TYPE_PREFIX`, `MAX_IDLE_CONNS`, `MAX_CONNS_PER_HOST`, `MAX_IDLE_CONNS_PER_HOST`, `IDLE_CONN_TIMEOUT` are missing. You may find them [here](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/pkg/env/nats_config.go#L10-L26).